specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json

```diff
@@ -0,0 +1,644 @@
+                "operationId": "56f30ceeeda5650db055a3c7",
+                "summary": "Detect Language",
+                "parameters": [{
+                    "name": "showStats",
+                    "in": "query",
+                    "description": "(optional) if set to true, response will contain input and document level statistics.",
+                    "type": "boolean"
+                }, {
+                    "name": "languageBatchInput",
+                    "in": "body",
+                    "schema": {
+                        "$ref": "#/definitions/LanguageBatchInput"
+                    },
+                    "description": "Collection of documents to analyze."
+                }],
+                "consumes": ["application/json", "text/json"],
+                "responses": {
+                    "200": {
+                        "description": "A successful call results in the detected language with the highest probability for each valid document",
+                        "schema": {
+                            "$ref": "#/definitions/LanguageBatchResult"
+                        }
+                    },
+                    "default": {
+                        "description": "Error Response",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Successful Detect Language request": {
+                    "$ref": "./examples//SuccessfulLanguagesRequest.json"
+                    }
+                },
+                "deprecated": false,  
+                "produces": ["application/json", "text/json"]
+            }
+        },
+        "/entities": {
+            "post": {
+              "tags": [
+                "Entities"
+              ],
+                "description": "The API returns a list of known entities and general named entities (\\\"Person\\\", \\\"Location\\\", \\\"Organization\\\" etc) in a given document. Known entities are returned with Wikipedia Id and Wikipedia link, and also Bing Id which can be used in Bing Entity Search API. General named entities are returned with entity types. If a general named entity is also a known entity, then all information regarding it (Wikipedia Id, Bing Id, entity type etc) will be returned. See the <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-entity-linking#supported-types-for-named-entity-recognition\">Supported Entity Types in Text Analytics API</a> for the list of supported Entity Types. See the <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/text-analytics-supported-languages\">Supported languages in Text Analytics API</a> for the list of enabled languages.\n",
```

Comment:
`\\\"Organization\\\"`, you can just use
```
`Organization`
```
Easier to read as json and better display in auto-generated documents.

Reply:
Updated the description to be consistent with previous versions